cli/mysql.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/cheggaaa/pb"
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/docker/docker/pkg/term"
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/controller/client"
+	ct "github.com/flynn/flynn/controller/types"
+)
+
+func init() {
+	register("mysql", runMysql, `
+usage: flynn mysql console [--] [<argument>...]
+       flynn mysql dump [-q] [-f <file>]
+       flynn mysql restore [-q] [-f <file>]
+
+Options:
+	-f, --file=<file>  name of dump file
+	-q, --quiet        don't print progress
+
+Commands:
+	console  Open a console to a Flynn mysql database. Any valid arguments to mysql may be provided.
```

Comment:
All valid points, but I still don't see why not just call it "MariaDB" if it is a MariaDB. :) This would also hint the users to the additional features of Maria.
And the name "MySQL" wouldn't be taken on the server side? Assuming the Flynn team or someone from the outer side wants to implement support for the original MySQL DB.